scripts/check-coding-style.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
-script_dir=$(dirname "$(readlink -f "$0")")
-
+READLINK=$(which greadlink)
+[ -z "$READLINK" ] && {
+	READLINK=$(which readlink)
+}
+[ -z "$1" ] && {
+	echo "USAGE: $0 FILE"
+	exit 1
+}
+CHECKSOURCE=$($READLINK -f "$1")
```

Comment:
Minor update, the latest count is 3 files, complying to kernel coding style ;-) However, I changed to behavior of the script, either pass a single file to be checked as a parameter OR provide a list of files in ```scripts/files.txt```. This is also add to travis config now.

Hope you (@fho) like that better?!